CONTRIBUTING.md

```diff
@@ -96,7 +96,7 @@ Here are some bad examples of clang-format-ing:
 ```
 @@ -58,8 +59,7 @@ static int register_membarriers(void)
          }
- 
+
```

Comment:
This " " space at the end of line is intended to be there as it is a part of diff format.